prov/efa/src/rxr/rxr_init.c

```diff
@@ -447,6 +447,7 @@ static int rxr_info_to_rxr(uint32_t version, const struct fi_info *core_info,
 		if (hints->domain_attr && hints->domain_attr->mr_mode & FI_MR_LOCAL)
 			info->domain_attr->mr_mode |= FI_MR_LOCAL;
 
+		info->domain_attr->resource_mgmt = hints->domain_attr->resource_mgmt;
```

Comment:
`hints` or `hints->domain_attr` may be NULL so this needs to be checked. If this isn't specified we should set it to `FI_RM_ENABLED`.